packages/hash/api/src/model/index.ts

```diff
@@ -30,6 +31,9 @@ export { PropertyTypeModel };
 export * from "./ontology/link-type.model";
 export { LinkTypeModel };
 
+export * from "./ontology/entity-type.model";
+export { EntityTypeModel };
+
 export * from "./entityType.model";
 export { EntityType };
```

Comment:
Having these here is pretty confusing and I don't see a mention of it anywhere. Are we leaving it to avoid migrating too much for now and will delete in a bit?

Reply:
Correct. Deprecating these will be a lot of work, and we reasoned it would ideally be done in standalone PRs instead of this one so that it doesn't block other tasks (such as creating users in the datastore using the new model classes).

Reply:
I agree with the reasoning :) can we add a TODO?

Reply:
Sure, have done so in https://github.com/hashintel/hash/pull/903/commits/b43f5697f330ea6e90deea085bb3615b5b6a3135